### PR TITLE
Add plugin system

### PR DESCRIPTION
**Pull request by IamShobe**

### CODE REVIEW

File: src/elrados/backend/main.py [hunk 1]
```diff
@@ -36,9 +38,30 @@ def send_to_client(self, sender, instance, **kwargs):
...
+    """Read from rotest.yml config elrados segment."""
+    config_path = search_config_file()
+    if config_path is None:
+        configuration_content = None
+    else:
+        with open(config_path, "r") as config_file:
+            configuration_content = config_file.read()
+
+    yaml_configuration = yaml.load(configuration_content)
+    if "elrados" not in yaml_configuration:
```
Comment (gregoil) [inline src/elrados/backend/main.py]: Just use ```get``` with default value

File: src/elrados/backend/main.py [hunk 2]
```diff
@@ -36,9 +38,30 @@ def send_to_client(self, sender, instance, **kwargs):
...
+        return AttrDict({})
+
+    yaml_configuration = yaml_configuration["elrados"]
```
Comment (gregoil) [inline src/elrados/backend/main.py]: It returns ```AttrDict``` either way, no?

File: src/elrados/backend/main.py [hunk 3]
```diff
@@ -47,9 +70,13 @@ def setup_server():
             if resource.DATA_CLASS not in (None, NotImplemented)
             and issubclass(resource.DATA_CLASS, ResourceData))
 
+    default_resource = config.get("default_resource", None)
+    default_resource = resource_models[-1].__name__ \
+        if default_resource is None else default_resource
```
Comment (gregoil) [inline src/elrados/backend/main.py]: Standard ```if``` section might look better here

**Review by gregoil — CHANGES_REQUESTED** [review]
Add documentation for the new additions in the README. 
Also I'd like to see how the declaration of plugins looks

File: src/elrados/backend/main.py (hunk 2 above)
Comment (IamShobe) [reply src/elrados/backend/main.py]: yes y?

**Review by gregoil — CHANGES_REQUESTED** [review]
What about the README file?

**Review by gregoil — APPROVED** [review]

**Pull request merged**